checker/src/main/java/org/checkerframework/checker/nullness/collection-object-parameters-may-be-null.astub

```diff
@@ -0,0 +1,152 @@
+// For documentation, see
+// https://checkerframework.org/manual/#collection-object-parameters-may-be-null .
+
+// The affected methods are:
+// containsKey(Object)
+// contains(Object)
+// containsValue(Object)
+// get(Object)
+// getOrDefault(Object, V)
+// indexOf(Object)
+// lastIndexOf(Object)
+// removeFirstOccurrence(Object)
+// removeLastOccurrence(Object )
+// remove(Object)
+// remove(Object, Object)
+
+package java.util;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public abstract class AbstractList<E> extends AbstractCollection<E> implements List<E> {
+    public int indexOf(@Nullable Object o);
+    public int lastIndexOf(@Nullable Object o);
+}
+
+public class ArrayList<E> extends AbstractList<E>
```

Comment:
This class can be removed if the annotations on ArrayList in the annotated JDK are changed.

Reply:
I was wondering about this also. In fact, it looks like the annotations on `ArrayList` in the annotated JDK already match what's here.

Not that there's any harm in redeclaring them, especially if it's easier for you to generate or organize the stub file that way.

I think I saw some other classes that were technically redundant, like `PriorityQueue`.